ui/components/multichain/account-list-item-menu/account-list-item-menu.js

```diff
@@ -137,6 +144,21 @@ export const AccountListItemMenu = ({
     dispatch(updateAccountsList(updatedPinnedAccountList));
   };
 
+  const handleHidding = (address) => {
```

Comment:
Can we move the logic for `updatedHiddenAccountList` to the action instead?  The action would receive just the address, then do the filter accordingly?  That way, our component stays "dumb" about the data and handles the minimal amount of data?

Reply:
Hmm..I think these actions are component-specific so we can keep them here. Can be done in the housekeeping as well. Wdyt?